server/routes/posts.js

```diff
@@ -45,7 +44,7 @@ router.post('/', (req, res) => {
       title,
       customer_id: parseCustId,
       //post_status_id: parsePostStatId, -- will uncomment when post_statuses table is seeded
-     // post_priority_id: parsePostPriorId, -- will uncomment when post_priorities table is seeded
+      // post_priority_id: parsePostPriorId, -- will uncomment when post_priorities table is seeded
```

Comment:
Try to make sure files like this aren't added to your PR. It functionally adds nothing but makes it more difficult for the reviewer to understand what's going on.